common/src/main/java/us/ilite/common/lib/control/PIDGains.java

```diff
@@ -0,0 +1,19 @@
+package us.ilite.common.lib.control;
+
+public class PIDGains {
+    public final double mP;
```

Comment:
Since these are `final`, they are constants and should be prefixed with `k` instead of `m`. See https://github.com/iliterobotics/FRC-Robot-2019#conventions.